tests/PHPUnit/Integration/DataAccess/ModelTest.php

```diff
@@ -687,6 +687,23 @@ public function testDeleteInvalidationsForDeletedSites()
         ], $invalidations);
     }
 
+    public function testGetInvalidationsInProgress()
+    {
+        $this->insertInvalidations([
+            ['idsite' => 1, 'date1' => '2014-02-01', 'date2' => '2014-02-28', 'period' => 2, 'name' => 'done'],
```

Comment:
Can we maybe try to use the constants in the tests here and in QueueConsumerTests, at least for the newly added tests?

```suggestion
            ['idsite' => 1, 'date1' => '2014-02-01', 'date2' => '2014-02-28', 'period' => Month::PERIOD_ID, 'name' => 'done'],
```

Reply:
Same as below, not critical. I understand it would be a big undertaking to try and update the period IDs even within the code changed in this PR. It might be good to start at some point, though :)